spirv_reflect.c

```diff
@@ -3439,6 +3472,44 @@ static SpvReflectResult ParseByteAddressBuffer(SpvReflectPrvParser* p_parser, Sp
   return SPV_REFLECT_RESULT_SUCCESS;
 }
 
+static SpvReflectResult ParseFunctionParameterAccess(SpvReflectPrvParser* p_parser, uint32_t callee_function_id,
+                                                     uint32_t function_parameter_index, uint32_t* accessed) {
```

Comment:
Function params and vars that are pointers should have `p_` prefix so `accessed` should be `p_accessed`.

Reply:
This has been fixed.